examples/zkapps/09-recursion/src/Add.ts

```diff
@@ -90,4 +43,29 @@ const Add = ZkProgram({
     },
   },
 });
+
+
+async function main() {
+  console.log('compiling...');
+
+  const { verificationKey } = await Add.compile();
+
+  console.log('making proof 0');
+
+  const proof0 = await Add.init(Field(0));
+
+  console.log('making proof 1');
+
+  const proof1 = await Add.addNumber(Field(4), proof0, Field(4));
+
+  console.log('making proof 2');
+
+  const proof2 = await Add.add(Field(4), proof1, proof0);
+
+  console.log('verifying proof 2');
+  console.log('proof 2 data', proof2.publicInput.toString());
+
+  const ok = await verify(proof2.toJSON(), verificationKey);
```

Comment:
toJSON spits out an outject, may not do anything

Reply:
Yes, it only return true or false.